tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -33,7 +34,7 @@ describe(__filename, () => {
         {...props}
       />,
       AddonMoreInfoBase
-    );
+    ).find(DefinitionList).shallow();
```

Comment:
I think this is a pattern we should avoid because it involves reaching into the implementation of another component. The reason you had to do this was because you used `<DefinitionList items={...} />` instead of rendering children elements. If you switch to children elements then you won't need to dive into `<DefinitionList />`.

I took a quick stab at it and I think it will be possible with children elements. How about something more like this?

```diff
diff --git a/src/amo/components/AddonMoreInfo/index.js b/src/amo/components/AddonMoreInfo/index.js
index 0ae98231b..804a3bb0c 100644
--- a/src/amo/components/AddonMoreInfo/index.js
+++ b/src/amo/components/AddonMoreInfo/index.js
@@ -171,69 +171,30 @@ export class AddonMoreInfoBase extends React.Component<Props> {
   }: Object) {
     const { i18n } = this.props;
     return (
-      <DefinitionList
-        className="AddonMoreInfo-dl"
-        items={[
-          {
-            dt: i18n.gettext('Add-on Links'),
-            dd: (
-              <ul className="AddonMoreInfo-links-contents-list">
-                {homepage}
-                {supportUrl}
-                {supportEmail}
-              </ul>
-            ),
-            key: 'AddonMoreInfo-links',
-            renderIf: homepage || supportUrl || supportEmail,
-          },
-          {
-            dt: i18n.gettext('Version'),
-            dd: version,
-            key: 'AddonMoreInfo-version',
-            renderIf: version,
-          },
-          {
-            dt: i18n.gettext('versionLastUpdated'),
-            dd: versionLastUpdated,
-            key: 'AddonMoreInfo-last-updated',
-          },
-          {
-            dt: i18n.gettext('License'),
-            dd: versionLicenseLink,
-            key: 'AddonMoreInfo-license',
-            renderIf: versionLicenseLink,
-          },
-          {
-            dt: i18n.gettext('Privacy Policy'),
-            dd: privacyPolicyLink,
-            key: 'AddonMoreInfo-privacy-policy',
-            renderIf: privacyPolicyLink,
-          },
-          {
-            dt: i18n.gettext('End-User License Agreement'),
-            dd: eulaLink,
-            key: 'AddonMoreInfo-eula',
-            renderIf: eulaLink,
-          },
-          {
-            dt: i18n.gettext('Version History'),
-            dd: (
-              <ul className="AddonMoreInfo-links-contents-list">
-                {versionHistoryLink}
-                {betaVersionsLink}
-              </ul>
-            ),
-            key: 'AddonMoreInfo-version-history',
-            renderIf: versionHistoryLink || betaVersionsLink,
-          },
-          {
-            dt: i18n.gettext('Usage Statistics'),
-            dd: statsLink,
-            key: 'AddonMoreInfo-stats',
-            renderIf: statsLink,
-          },
-        ]}
-      />
+      <DefinitionList className="AddonMoreInfo-dl">
+        {(homepage || supportUrl || supportEmail) && (
+          <Definition term={i18n.gettext('Add-on Links')}>
+            <ul className="AddonMoreInfo-links-contents-list">
+              {homepage}
+              {supportUrl}
+              {supportEmail}
+            </ul>
+          </Definition>
+        )}
+        {version && (
+          <Definition term={i18n.gettext('Version')}>
+            {version}
+          </Definition>
+        )}
+        <Definition term={i18n.gettext('versionLastUpdated')}>
+          {versionLastUpdated}
+        </Definition>
+        {versionLicenseLink && (
+          <Definition term={i18n.gettext('License')}>
+            {versionLicenseLink}
+          </Definition>
+        )}
+      </DefinitionList>
     );
   }
 
```

Also, I don't think we need `renderNodesIf()` anymore. I can't remember exactly why we added it but I think it was maybe just a workaround for not having `React.Fragment`.

Reply:
Oh damn, that's a nice API. It means having to import `DefinitionList` and `Definition[Item]` but I like it. I thought about using children before but my approach was still too programmatic. I'll do that. 👍